<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 490 35px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);
</string>
   </property>
   <property name="text">
    <string>鼎菁科技</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 95px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>11:02</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="data">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 28px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>2018/02/26</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="week">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  450 30px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>周一</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="weather">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:  450 28px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>晴 28℃</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_login">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>480</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>set</string>
   </property>
  </widget>
  <widget class="QLabel" name="oneState">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 490 30px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>one</string>
   </property>
  </widget>
  <widget class="QLabel" name="twoState">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 490 30px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>two</string>
   </property>
  </widget>
  <widget class="QLabel" name="threeState">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 490 20px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>three</string>
   </property>
  </widget>
  <widget class="QLabel" name="signState">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 490 30px &quot;Bahnschrift&quot;;
color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>sign</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
